packages/Metric/src/Metric.tsx

```diff
@@ -106,7 +108,7 @@ const Metric: React.FunctionComponent<MetricProps> = ({
                     >
                         {tooltip && appearance !== "selected" && (
                             <Tooltip content={tooltip}>
-                                {getBrand() === "workleap" ? (
+                                {isWorkleap ? (
                                     <Help size="small" />) : <HelpSolid size="small" />}
```

Comment:
workleap icon should be "sm"

Reply:
None of these are Workleap icon though

Reply:
I only changed the conditional statement while at it. Help and helpsolid are both Igloo icons